docs/interactive/comments/utterances.md

```diff
@@ -6,15 +6,33 @@ Utterances is a commenting engine built on top of GitHub issues. It embeds a com
 Utterances is activated on this page. You can see the comment box at the bottom of this page's content. Click the "log in" button and you'll be able to post comments!
 ```
 
+To enable comments using Utterances you need to do the following: 
+1. Make repository is public so that others can comment. 
+1. Install the [utterances app](https://github.com/apps/utterances) on the repo. 
+1. Add the following snippet at the bottom of the page. This HTML activates utteranc.es on a specific page. 
+
+```html
```

Comment:
This should automatically be added to your page though, see: https://github.com/executablebooks/sphinx-comments/blob/dd90fed156b8d7f65b6b58245056a92e469ef3de/sphinx_comments/__init__.py#L68

Do you not see it in your source code, or perhaps you do have JavaScript active?
Otherwise, this should most probably be raised as an issue on https://github.com/executablebooks/sphinx-comments

Reply:
Dropped the snippet part of the suggestion